lib/kramdown_rpf/rpf.rb

```diff
@@ -112,20 +121,43 @@ def self.convert_print_only_to_html(content)
         ::Kramdown::Document.new("<div class=\"u-print-only\">\n#{content}</div>", KRAMDOWN_OPTIONS).to_html
```

Comment:
...but this is not changed here

Reply:
food for thought - https://anti-pattern.com/always-use-double-quoted-strings-in-ruby

Reply:
That one interpolates a variable, so not changed.
Rubocop flags one, but not t'other